packages/ckeditor5-list/src/listediting.js

```diff
@@ -168,7 +169,7 @@ export default class ListEditing extends Plugin {
 
 			data.preventDefault();
 			evt.stop();
-		}, { priority: 'high' } );
```

Comment:
Why? It's useful if such quirks are documented in the code.

Reply:
I added a code comment.